data/json/itemgroups/clothing.json

```diff
@@ -188,5 +188,90 @@
       [ "pocketwatch", 25 ],
       [ "wristwatch", 100 ]
     ]
+  },
+  {
+    "id": "clothing_technician",
```

Comment:
This group is redundant. You can push this to `clothing_work_set` which includes the `jumpsuit`